VK_V1.0/VK_V1.0/View/LikeControl.swift

```diff
@@ -0,0 +1,58 @@
+// LikeControl.swift
+
+import UIKit
+
+/// Компонент для отображения количества лайков
+final class LikeControl: UIControl {
+    // MARK: - Private enum
+
+    private enum Constants {
+        static let heartImageName = "suit.heart"
+        static let fillHeartImageName = "suit.heart.fill"
+    }
+
+    // MARK: - Private IBOutlets
+
+    @IBOutlet private var heartImageView: UIImageView!
+    @IBOutlet private var countLikeLabel: UILabel!
+
+    // MARK: - Private property
+
+    private var isLike = false
+    private var touches = 0 {
+        didSet {
+            countLikeLabel.text = "\(touches)"
+        }
+    }
+
+    // MARK: - LifeCycle
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        createGestureRecognizer()
+    }
+
+    // MARK: - Private methods
+
+    private func createGestureRecognizer() {
+        let tap = UITapGestureRecognizer(target: self, action: #selector(handleTapAction(_:)))
```

Comment:
```suggestion
        let tapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(handleTapAction(_:)))
```
так же, возможно попросят поменять